server-common/src/main/java/org/apache/kafka/server/common/GroupVersion.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.server.common;
+
+import java.util.Collections;
+import java.util.Map;
+
+public enum GroupVersion implements FeatureVersion {
+
+    // Version 1 enables the classic rebalance protocol. This is the default
+    // behavior even if the feature flag is not set.
+    GV_1(1, MetadataVersion.IBP_3_8_IV0, Collections.emptyMap()),
```

Comment:
When we bootstrap we technically will not get this version. Does that make sense? I'm wondering if the metadata version should be the bootstrap metadata version since it's a little odd to say the classic protocol is not set by default.

Alternatively we take the version 0 approach.

Reply:
Updated to use the version 0 approach.